test/lib/ufe/testUIIcons.py

```diff
@@ -85,9 +85,7 @@ def testUIIcons(self):
         # All the prim types to test
         usdVer = Usd.GetVersion()
         primTypes = [
-                # Prim Type                # Icon file name
-            ('ALExamplePolyCubeNode',   'out_USD_UsdTyped.png'),
-            ('ALFrameRange',            'out_USD_UsdTyped.png'),
+            # Prim Type                 # Icon file name
             ('ALMayaReference',         'out_USD_MayaReference.png'),
```

Comment:
What about the ALMayaReference? Should it be moved too?

Reply:
The schema for that one is in `lib`, so its always there.